services/app/src/pages/game/Game.jsx

```diff
@@ -33,6 +35,17 @@ function Room() {
     );
   };
 
+  const skipMove = (userId) => {
+    if (userId && ws.readyState === wsReadyStates.OPEN) {
```

Comment:
We probably want to have this `ws.readyState` check for all / most of the events we are sending to the backend, so I think it would be a good idea to introduce a wrapper arround `ws.send` which would check the `readyState` and then pass the request to the `ws.send`.

Reply:
It will be done on the refactor task. On this one was focusing only on the skip player feature and wasn't applying scout boy principles.